### PR TITLE
Queue

### PR DESCRIPTION
**Pull request by shakalee14**

### CODE REVIEW

File: spec/queue.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  it('exists', () => {
+    expect(Queue).to.be.a('function')
+  })
+
+  context('enqueue', () => {
+    it('adds an element to the back of the queue', () => {
+      const bankLine = new Queue()
+
+      bankLine.enqueue('customer one')
+      expect(bankLine.length()).to.equal(1)
```
Comment (rae-ralston) [inline spec/queue.js:19]: add a 'customer one' stipulation?

File: src/queue.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+      this.lengthVal --
+      return temp
+    } else {
+      return null
+    }
+  }
+
+  front(){
+    if( this.lengthVal > 0 ){
```
Comment (rae-ralston) [inline src/queue.js]: ternary?

File: src/queue.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    } else {
+      return null
+    }
+  }
+
+  isEmpty(){
+    return this.lengthVal <= 0
+  }
+
+  length(){
```
Comment (rae-ralston) [inline src/queue.js:57]: can you use es6 arrow functions on these ? or is that not a thing bc it's a class?

**Review by rae-ralston — COMMENTED** [review]
<3

File: src/queue.js (hunk 3 above)
Comment (lumodon) [reply src/queue.js:57]: Looks like that's an experimental feature of babel not fully implemented in ES6 yet. Maybe ES7.

**Pull request merged**